onert-micro/onert-micro/include/train/tests/numbers_classification_task/data/train_input.h

```diff
@@ -0,0 +1,1250 @@
+  0xff, 0xfe, 0x7e, 0x3f, 0xff, 0xfe, 0x7e, 0x3f, 0xdd, 0xdc, 0xdc, 0x3e, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xd0, 0xcf, 0x4f, 0x3f, 0xff, 0xfe, 0x7e, 0x3f,
+  0xff, 0xfe, 0x7e, 0x3f, 0xdd, 0xdc, 0xdc, 0x3e, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0xd0, 0xcf, 0x4f, 0x3f, 0xff, 0xfe, 0x7e, 0x3f, 0xff, 0xfe, 0x7e, 0x3f,
+  0xdd, 0xdc, 0xdc, 0x3e, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0xd0, 0xcf, 0x4f, 0x3f, 0xff, 0xfe, 0x7e, 0x3f, 0xf6, 0xf5, 0x75, 0x3f, 0x8f, 0x8e, 0x8e, 0x3e,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xd0, 0xcf, 0x4f, 0x3f,
+  0xff, 0xfe, 0x7e, 0x3f, 0xe4, 0xe3, 0x63, 0x3f, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x87, 0x86, 0x86, 0x3e, 0xf3, 0xf2, 0x72, 0x3f, 0xff, 0xfe, 0x7e, 0x3f,
+  0xe4, 0xe3, 0x63, 0x3f, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0xb5, 0xb4, 0x34, 0x3f, 0x00, 0x00, 0x80, 0x3f, 0xff, 0xfe, 0x7e, 0x3f, 0xd5, 0xd4, 0x54, 0x3f,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x81, 0x80, 0x80, 0x3c, 0xe6, 0xe5, 0x65, 0x3f,
+  0xff, 0xfe, 0x7e, 0x3f, 0xff, 0xfe, 0x7e, 0x3f, 0xbb, 0xba, 0xba, 0x3e, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0xdf, 0xde, 0xde, 0x3e, 0xff, 0xfe, 0x7e, 0x3f, 0xff, 0xfe, 0x7e, 0x3f,
+  0xff, 0xfe, 0x7e, 0x3f, 0xe4, 0xe3, 0x63, 0x3f, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0xdf, 0xde, 0xde, 0x3e, 0xff, 0xfe, 0x7e, 0x3f, 0xff, 0xfe, 0x7e, 0x3f, 0xff, 0xfe, 0x7e, 0x3f,
+  0xc1, 0xc0, 0xc0, 0x3e, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xdf, 0xde, 0xde, 0x3e,
+  0xff, 0xfe, 0x7e, 0x3f, 0xff, 0xfe, 0x7e, 0x3f, 0xe5, 0xe4, 0x64, 0x3f, 0xa5, 0xa4, 0x24, 0x3e,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+unsigned int numbers_classification_task_input_size = 19360;
```

Comment:
data size is too huge. Could you please reduce this like under 1k ?

Reply:
done